### DISCUSSION TITLE
Konat kelimesinin anlamı

### DISCUSSION THREAD

**Discussion by hi-rullah** [turkce-yazilim-konati/yazilimca#86] [Soru-Yanıt]
Türkçe Yazılım Konat'ını Twitter'da görmüştüm. Konat kelimisini de ilk defa burda duydum. TDK sözlükte konat kelimesini aradım karşılık bulunmadı.
Konat platform kelimesine karşılık kullanıldığını düşünüyorum. Kelime hakkında kaynak gösterilirse memnun olurum.
Saygılarımla,

**Comment by volkantash** [comment]
Bu güzel soru için teşekkür ederiz. Konat sözcüğü DLT'de karşımıza çıkan bir sözcüktür. Birbirlerine yanaşan, toplanan insan kümesi demektir.
https://youtube.com/clip/UgkxkxiUcfWc9sqNqf96FWvbfUobvqAkSXc9?si=NaHycarDRwV3Nsad

**Reply by hi-rullah** [reply]
Anlaşıldı, teşekkür ederim :)

**Reply by volkantash** [reply]
Rica ederim.
